website/blog/2024-10-23-release-0.76-new-architecture.md

```diff
@@ -0,0 +1,221 @@
+authors: [blakef, cipolleschi, frankcalise, gabrieldonadel]
+tags: [engineering]
+date: 2024-10-23
+---
+
+Today we are excited to release React Native 0.76! This is a major milestone for React Native, with the New Architecture enabled by default, and the new React Native DevTools. This has been the culmination of 6 years of hard work from our incredible community of developers to make this day happen.
+
+### Highlights
+
+- [React Native New Architecture by default](#react-native-new-architecture-by-default)
+- [React Native DevTools](#react-native-devtools)
+- [Box Shadow and Filter style props](#box-shadow-and-filter-style-props)
+
+### Breaking Changes
+
+- [Removed the dependency on the react-native-community/cli](#removed-the-dependency-on-the-react-native-communitycli)
+- [[Android] React Native C++ code is distributed as libreactnative.so](#android-react-native-is-distributed-as-a-single-library)
+- [Updates to Minimum iOS and Android SDK requirements](#updates-to-minimum-ios-and-android-sdk-requirements)
+
+<!--truncate-->
+
+## Highlights
+
+### React Native New Architecture by default
+
+The New Architecture is finally here and starting from 0.76 is enabled by default in your projects.
+
+Thanks to our collaboration with the whole community, we feel now confident to make this step. We verified that more than [880 libraries](https://reactnative.directory/?newArchitecture=true) are compatible with the New Architecture, including all the Expo Modules.
+
+To make sure your experience with the New Architecture will be great, we worked with [BlueSky](https://github.com/bluesky-social/social-app/releases/tag/1.92.0-na-rc.2) and Kraken who are already using it in production.
+
+This change will be completely transparent to you: by upgrading to version 0.76, following the instruction in the [upgrade helper](https://react-native-community.github.io/upgrade-helper/?from=0.75.4&to=0.76.0), you’ll also be turning on the New Architecture. We don’t expect any changes in behaviors in your app.
+
+If for any reason you can’t move to the New Architecture, you can still opt-out from it.
+
+On Android, you can modify the `gradle.properties` file and turn the `newArchEnabled` flag to `false`.
+
+```diff
+-newArchEnabled=false
++newArchEnabled=false
+```
+
+On iOS, you can opt-out from the New Architecture by installing Cocoapods dependencies with the command:
+
+```bash
+RCT_NEW_ARCH_ENABLED=0 bundle exec pod install
+```
+
+As part of the work to make the New Architecture the default, we also revamped most sections of the [website](https://reactnative.dev) to reflect these changes. If you need the documentation of the [Legacy Native Modules](https://reactnative.dev/docs/0.76/legacy/native-modules-intro) and [Legacy Native Components](https://reactnative.dev/docs/0.76/legacy/native-components-android), you can find it in the Legacy Architecture section. Otherwise, the rest of the website assumes that you are looking for New Architecture documentation.
```

Comment:
To avoid forcing internal links to open in new tab let's use relative links in blog post body:

```suggestion
As part of the work to make the New Architecture the default, we also revamped most sections of the website to reflect these changes. If you need the documentation of the [Legacy Native Modules](/docs/0.76/legacy/native-modules-intro) and [Legacy Native Components](/docs/0.76/legacy/native-components-android), you can find it in the Legacy Architecture section. Otherwise, the rest of the website assumes that you are looking for New Architecture documentation.
```

Linking back to the homepage feels a bit redundant, that's why I have also suggested skipping that anchor.